src/Components/JoinScreen/WaitingRoom.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactElement, useEffect, useState } from "react";
+
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import { Grid } from "@mui/material";
+import Typography from '@mui/material/Typography';
+import socket from "../Websocket/socket";
+
+type Player = {
+  username: string,
+  status: boolean
+} 
+const WaitingRoom = (): ReactElement => {
+  if(!window.location.search) return <div>No game found</div>
```

Comment:
Is this just if there is param after the ? in the search bar? Shouldn't we be checking if the value after the ? is a valid room instead? Cause then an empty string would also be an invalid room.